setup/instance/apply.go

```diff
@@ -0,0 +1,57 @@
+package instance
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"net/http"
+	"time"
+)
+
+func (l *LocalInstance) Apply(wait bool) error {
+	if err := l.assertExists(); err != nil {
```

Comment:
@sesh-kebab i assume you're getting this error because you didn't first run `l0-setup init`